components/layout/positioned.rs

```diff
@@ -709,6 +662,10 @@ impl HoistedAbsolutelyPositionedBox {
             )
             .with_specific_layout_info(specific_layout_info)
         };
+
+        // This is an absolutely positioned element, which means it also establishes a
+        // containing block for absolutes. We lay out any absolutely positioned children
+        // here and pass the rest to `hoisted_absolutes_from_children.`
```

Comment:
I think it would nice if the `layout_collected_children` function took `Absolute | Fixed` as a parameter. So that it was explicitly "layout collected position:absolute children"

Reply:
Perhaps even as a closure filter parameter?

Reply:
Oh, I think my response to this was lost. My hope with this change was to make the API impossible to misuse. Passing in a parameter here means that all the of the callers, which are scattered through layout, need to pass the correct parameter to this function, which means carefully getting the details from style. For now, I think it's better that this function just encapsulates all of the difficulty of doing the right thing.